VEDA_OS01/models.py

```diff
@@ -428,3 +428,31 @@ def __unicode__(self):
             self.status_email,
             self.file_complete
         )
+
+
+class TranscriptionProviderType(object):
+    """
+    3rd party transcript providers.
+    """
+
+    THREE_PLAY = '3PlayMedia'
+    CIELO24 = 'Cielo24'
+    CHOICES = (
+        (THREE_PLAY, THREE_PLAY),
+        (CIELO24, CIELO24),
+    )
+
+
+class TranscriptionProvider(models.Model):
+    """
+    Model to contain third party transcription service provider preferances.
+    """
+    org = models.CharField('Organization', max_length=50, unique=True)
+    provider = models.CharField('Transcription provider', max_length=50, choices=TranscriptionProviderType.CHOICES)
```

Comment:
@mushtaqak I think we need to make org & provider unique in a combination. So that orgs can have both credentials to use them in different courses.